### ISSUE TITLE
Ma Banque - Crédit Agricole

### ISSUE THREAD

**Issue by gamrix0** [akc3n/banking#73]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### App name

Ma Banque

### Link to app

https://play.google.com/store/apps/details?id=fr.creditagricole.androidapp&gl=FR

### App version

v29.0

### Country of the app

France

### Build Number

SP2A.220405.004.2022041600

### Device

- [ ] Pixel 6 Pro
- [X] Pixel 6
- [ ] Pixel 5a
- [ ] Pixel 5
- [ ] Pixel 4a (5G)
- [ ] Pixel 4a
- [ ] Pixel 4 XL
- [ ] Pixel 4
- [X] Pixel 3a XL
- [ ] Pixel 3a
- [ ] Pixel 3 XL
- [ ] Pixel 3

### Google Play installed

- [X] Installed
- [ ] Not installed

### Google Play services Network permission revoked?

- [ ] Revoked
- [X] Not revoked
- [ ] I did not have Google Play services installed

### SafetyNet Enforcement

- [ ] Enforced
- [ ] Not enforced
- [X] Unsure

### Native code debugging

- [X] Enabled
- [ ] Disabled

### Stock OS compatibility

- [X] Works
- [ ] Does not work
- [ ] Not tested

### Profile app tested in

- [X] Main user profile
- [ ] Secondary user profile
- [ ] I used a work profile device manager app (Please mention more details about it down below like the app)

### Description of the app's functionality

Logging works.
2FA works (Secure Pass).
Biometrics authentication works. 
Internal messaging works.

**Comment by akc3n** [comment]
Thanks @gamrix0 Will be adding this right now.

**Comment by Rathmox** [comment]
Update:

The app works almost completely without play services

Without Play Services, SécuriPass can't be activated

Once SécuriPass is activated, Play services can be removed entirely and everything will work.

I couldn't test entirely the part with mobile payements (which is another app), it just looks like the app refuses to work without Play Services.

**Comment by sjpp** [comment]
For the record, I'd like to add that I could not have it working in a secondary user profile.
I first tried to set it up in the owner profile (with sandboxed Play Services) and, as mentioned above it worked fine. Then I removed the app and the Play Services, set up a second user account, installed the Play Services and Ma Banque app, but could not managed to make it work. The app shows an error after message link validation and securiPass PIN code entry.

**Comment by plkrbb** [comment]
Hello, I installed "Ma banque"  v.42.0.0 on a Pixel 7 on my primary profile, with Google Play, network and sensors access, Native Code debugging Allowed, Exploit protection compatibility mode activated.

 I can't use it : it asks  to activate Securipass, it then sends me a message with a link to use, and then the app tells "an error occured, try later" .
Thanks you all for any hints.

**Comment by romainreignier** [comment]
I have installed "Ma Banque" version 45.1.0-345 on a Pixel 9a, in the only user profile I have.
Without the Google Play Services installed, the SecuriPass process failed at the end, even with the link in the SMS correctly opening the app.
But with the Google Play Services installed, I could enable the SecuriPass on the Phone and uninstall the Play Services after.

**Comment by Elu43** [comment]
Hi
I can use Ma Banque but Paiement mobile doesn't work anymore, is it the same for you?

**Comment by Alerymin** [comment]
Paiement Mobile never worked for me as it required play services

But the app does not work anymore, but it fluctuates

**Comment by GhostFRR** [comment]
The last version of Paiement mobile doesnt work. But if you install the 8.3.16, it's work. You can do that with aurora store and use manual install.

**Comment by Elu43** [comment]
> The last version of Paiement mobile doesnt work. But if you install the 8.3.16, it's work. You can do that with aurora store and use manual install.

Indeed it works for me now, until it stops being supported...

Thanks !

**Comment by GhostFRR** [comment]
I don't know if it's a support issue or a bug. Previously, I had a particularly good setup with a rooted LineageOS. And I managed to pass the Revolut tests, which are known for being difficult. Furthermore, Credit Agricole, in none of its applications, has taken any measures against custom ROMs and/or rooted devices, at least to my knowledge. What's more, I saw a lot of negative reviews when the update was released, and I don't think there are that many people on custom ROMs ! We'll see if things improve with the next update.
